controllers/admin/AdminImportController.php

```diff
@@ -3146,15 +3164,26 @@ protected function addressImportOne($info, $force_ids, $validateOnly = false)
                 $customer_list = Customer::getCustomersByEmail($customer->email);
 
                 if (count($customer_list) == 0) {
-                    $this->errors[] = sprintf(
-                        Tools::displayError('%1$s does not exist in database %2$s (ID: %3$s), and therefore cannot be '.($validateOnly?'validated':'saved')),
-                        Db::getInstance()->getMsgError(),
-                        $customer->email,
-                        (int)$address->id_customer
+                    $this->errors[] = $this->trans(
+                        '%1$s does not exist in database %2$s (ID: %3$s), and therefore cannot be %4$s',
+                        array(
+                            Db::getInstance()->getMsgError(),
+                            $customer->email,
+                            (int)$address->id_customer,
+                            ($validateOnly?'validated':'saved')
+                        ),
+                        'Admin.Advparameters.Notification'
                     );
                 }
             } else {
-                $this->errors[] = sprintf(Tools::displayError('The customer ID #%d does not exist in the database, and therefore cannot be '.($validateOnly?'validated':'saved')), $address->id_customer);
+                $this->errors[] = $this->trans(
+                    'The customer ID #%d does not exist in the database, and therefore cannot be %2$s',
```

Comment:
Again, this should also be split into two strings:
* `The customer ID #%d does not exist in the database, and therefore cannot be validated`
* `The customer ID #%d does not exist in the database, and therefore cannot be saved`